docs/product/components/expandable.html

```diff
@@ -153,10 +153,10 @@
                     </div>
                 </div>
                 <div class="flex--item">
-                    <div class="d-flex gs8 gsx">
+                    <div class="d-flex gy8">
```

Comment:
## Change requested

Since `<label>` elements are rendered as `display: table`, the gap property doesn't apply. Any `<label>` with parents that include a gap property need either to have a margin added or have their `display` property altered to honor `gap`.